src/api/app/views/webui2/webui/user/_breadcrumb_items.html.haml

```diff
@@ -3,6 +3,14 @@
   = render partial: 'webui/configuration/breadcrumb_items'
   %li.breadcrumb-item.active{ 'aria-current' => 'page' }
     Manage Users
+- elsif current_page?(user_register_user_path)
+  = render partial: 'webui/configuration/breadcrumb_items'
+  %li.breadcrumb-item.active{ 'aria-current' => 'page' }
+    Create User
```

Comment:
The breadcrumbs are missing one element. Right now, it is `Home(link) / Configuration(link) / Create User(plain text)`, but it should be `Home(link) / Configuration(link) / Manage Users(link) / Create User(plain text)`